in foo
As check failed: an int (499) is not a C.
  0: main                      tests/negative/type-local4-test.toit:13:9
